packages/nuget/Sentry.Maui/3.18.0-preview.1.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "Sentry.Maui",
+  "canonical": "nuget:Sentry.Maui",
+  "version": "3.18.0-preview.1",
+  "package_url": "https://www.nuget.org/packages/Sentry.Maui",
+  "repo_url": "https://github.com/getsentry/sentry-dotnet",
+  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/maui"
```

Docs will land as 'guides' e.g: https://docs.sentry.io/platforms/dotnet/guides/aspnet/

```suggestion
  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/guides/maui/"
```

---

There's a redirect, and our other packages use it, so I was copying the pattern - but i'm fine either way.